manifests/mod/suphp.pp

```diff
@@ -6,7 +6,7 @@
 class apache::mod::suphp (
 ) {
   if  ($facts['os']['name'] == 'Ubuntu' and versioncmp($facts['os']['release']['major'], '15.10') >= 0) or
-  ($facts['os']['name'] == 'Debian' and versioncmp($::operatingsystemrelease, '8') >= 0) {
+  $facts['os']['family'] == 'Debian' {
```

Comment:
I now reread this and this looks like a bad rebase from my side. This also drops Ubuntu < 15.10 support here.

Reply:
https://github.com/puppetlabs/puppetlabs-apache/pull/2193 should fix that.

Reply:
Thank you @ekohl for the fixes.